jcl/src/java.base/share/classes/java/lang/ref/ReferenceQueue.java

```diff
@@ -195,13 +195,13 @@ public Reference<? extends T> remove(long timeout) throws IllegalArgumentExcepti
  *					true if reference is enqueued.
  *					false if reference failed to enqueue.
  */
-boolean enqueue(Reference<? extends T> reference) {
+void enqueue(Reference<? extends T> reference) {
 	/*[PR CMVC 96472] deadlock loading sun.misc.Cleaner */
 	/*[PR 102259] call Cleaner.clean(), do not enqueue */
 	if (reference instanceof Cleaner) {
 		reference.dequeue();
 		((Cleaner)reference).clean();
-		return true;
+		reference.setEnqueued();
```

Comment:
I suspect this, and the other a few lines below, should be removed, but it's what was done previously.

Reply:
I don't think we want this call to `setEnqueued()`, nor the one on line 213.